packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -185,6 +185,14 @@ export async function createSecretInSecretManagerIfNotExists(
   }
 }
 
+export async function deleteSecretFromSecretManager(secretId: string) {
```

Comment:
I would add a prompt to confirm deletion to lower the chances for accidents.

Reply:
Missed updating this one: I would argue it's not enough because that is at the very beginning of the operation. Adding a prompt here doesn't save everything, as deleting the DB and other things could also be qualified as "accidents", but I expect secrets to be more locked down than other resources for security. Not a blocker for deployment though, just a strong preference.